sources/api/netdog/src/wicked/mod.rs

```diff
@@ -185,9 +185,9 @@ mod tests {
                 assert_eq!(
                     expected.trim(),
                     generated,
-                    "failed test for net config version: '{}', interface: '{}'",
+                    "failed test for net config version: '{}', interface: '{:?}'",
                     version,
-                    interface.name.to_string()
+                    interface.name
```

Comment:
It's weird that clippy would suggest switching from `Display` to `Debug` format here. What was the actual warning?

Reply:
`clippy` was upset that `interface.name` should already implement DISPLAY without being converted to string, and suggested changing it to `*interface.name` instead. That suggestion ends up triggering another error as the size can't be known at compilation time. I figured `{:?}` was slightly less ugly than adding a `clippy` exception, but I could go either way.